spylib/webhook/__init__.py

```diff
@@ -23,7 +24,9 @@ class WebhookCreate(Enum):
     PUB_SUB = 'pubSubWebhookSubscriptionCreate'
 
 
-def validate(data: str, hmac_header: str, api_secret_key: str) -> bool:
+def validate(data: Union[str, bytes], hmac_header: str, api_secret_key: str) -> bool:
+    if isinstance(data, bytes):
+        data = data.decode(UTF8ENCODING)
     try:
         validate_hmac(secret=api_secret_key, sent_hmac=hmac_header, message=data, use_base64=True)
```

Comment:
Let's be safe and let's not touch the argument
```suggestion
    if isinstance(data, bytes):
        data_str: str = data.decode(UTF8ENCODING)
    else:
        data_str: str = data
    try:
        validate_hmac(secret=api_secret_key, sent_hmac=hmac_header, message=data_str, use_base64=True)
```